app/components/TumourSummaryEdit/index.tsx

```diff
@@ -323,32 +339,27 @@ const TumourSummaryEdit = ({
     return null;
   }, [handleClicked, handleDelete, handleKeyDown, newMicrobialData]);
 
-  const mutBurDataSection = useMemo(() => {
-    if (newMutationBurdenData) {
-      return (
-        <TextField
-          className="tumour-dialog__text-field"
-          label="Mutation Burden (Mut/Mb)"
-          value={newMutationBurdenData.totalMutationsPerMb}
-          name="totalMutationsPerMb"
-          onChange={handleMutationBurdenChange}
-          variant="outlined"
-          fullWidth
-          type="number"
-        />
-      );
-    }
-    return null;
-  }, [newMutationBurdenData, handleMutationBurdenChange]);
+  const mutBurDataSection = (
+    <TextField
+      className="tumour-dialog__text-field"
+      label="Mutation Burden (Mut/Mb)"
+      value={newMutationBurdenData?.totalMutationsPerMb ?? 0}
+      name="totalMutationsPerMb"
+      onChange={handleMutationBurdenChange}
+      variant="outlined"
+      fullWidth
+      type="number"
+    />
+  );
 
   const tmburMutBurSection = useMemo(() => {
-    if (newTmburMutData) {
+    if (reportType === 'rapid') {
```

Comment:
Does this mean that these fields will not be editable for Genomic reports anymore?

Reply:
I believe your open PR takes care of this, which should remove this check